harvardcards/static/js/modules/collections-index.js

```diff
@@ -1,11 +1,30 @@
-define(['jquery', 'components/slider/CollectionSlider', 'utils/utils'], function($, CollectionSlider, utils) {
+define(['jquery', 'components/slider/CollectionSlider', 'utils/utils', 'jqueryui'], function($, CollectionSlider, utils) {
 	return {
 		initModule: function(){
-			$('.slider').each(function(index, el) {
-				var slider = new CollectionSlider(el);
-			});
 
-			utils.setupConfirm();
+
+            $('.courseHeader').click(function() {
```

Comment:
This function should be pulled out and parameterized (i.e. pass in the target element) so that it can be called directly, rather than being triggered by a "click" event. That will also have the benefit of making it more testable.